src/test/java/com/github/nylle/javafixture/testobjects/factorymethod/FactoryMethodWithGenericArgument.java

```diff
@@ -9,7 +9,7 @@ private FactoryMethodWithGenericArgument(T value) {
     }
 
     public static <T> FactoryMethodWithGenericArgument<T> factoryMethod(T value) {
-        return new FactoryMethodWithGenericArgument(value);
+        return new FactoryMethodWithGenericArgument(42);
     }
```

Comment:
Is this intended?

Reply:
In this particular case, yes. This way I can assert that the factory method gets called and that we did *not* set a random value via reflection.

Reply:
I see. In that case, let's merge! 👍